### PR TITLE
don't impose CLASSIC model (issue #6)

### PR DESCRIPTION
**Pull request by jswhit**

### CODE REVIEW

**Comment by jswhit** [comment]
test now includes a small template file with no data

File: tests/test_ncio.f90 [hunk 1]
```diff
@@ -16,19 +15,48 @@ program test_ncio
...
-  print *,'natts=',dset%natts
-  do ndim=1,dset%ndims
-     print *,'dim',ndim,trim(dset%dimensions(ndim)%name),dset%dimensions(ndim)%len,dset%dimensions(ndim)%isunlimited
-  enddo
-  do nvar=1,dset%nvars
-     print *,'var',nvar,trim(dset%variables(nvar)%name),dset%variables(nvar)%ndims,dset%variables(nvar)%dimlens
+  dsetin = open_dataset('dynf000_template.nc')
+  ! create a copy of the template file
+  dset = create_dataset('dynf000.nc',dsetin)
+  print *,'ncid=',dsetin%ncid
```
Comment (edwardhartnett) [inline tests/test_ncio.f90]: What's with all the print statements?

Ideally tests will be terse. I usually write my tests to output something like:

*** Testing handling of some feature.
*** Testing ABC... ok
*** Testing XYZ... ok
SUCCESS!

The test should be testing all values internally, not printing them out for the user to see.

**Comment by jswhit** [comment]
@edwardhartnett this is ready for review now

**Comment by jswhit** [comment]
closes issue #5 and issue #6

**Review by edwardhartnett — CHANGES_REQUESTED** [review]

File: tests/test_ncio.f90 (hunk 1 above)
Comment (jswhit) [reply tests/test_ncio.f90]: fixed now

**Review by edwardhartnett — APPROVED** [review]
Awesome!

**Comment by jswhit** [comment]
Done with this now, should be ready to merge.

**Pull request merged**